golang/wasm/wrapper_js.go

```diff
@@ -208,6 +209,34 @@ func checkKeyStatus(_ js.Value, args []js.Value) any {
 	return promiseConstructor.New(handler)
 }
 
+func findDOHEndpoint(_ js.Value, args []js.Value) any {
+	if len(args) != 1 {
+		return "expected 1 argument: domainName"
```

Comment:
```suggestion
        // TODO: return rejected promise
		return "expected 1 argument: domainName"
```

strictly speaking these error cases should return a rejected promise, otherwise it's really awkward to consume as an API.

buttt since we are not doing this elsewhere yet I won't make you clean it up now.